### ISSUE TITLE
amp-bind fails to upgrade in Friendly Iframe

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#9395]
When `amp-bind` is used in an A4A ad, it fails when ad is rendered in FIE, works in 3p.

in FIE case, error is:

```
amp-ads.firebaseapp.com/dist/amp.js:12415 Uncaught (in promise) Error: amp-bind is not upgraded yet
    at Log.assert (amp-ads.firebaseapp.com/dist/amp.js:12415)
    at Object.upgradeElementInChildWindow (amp-ads.firebaseapp.com/dist/amp.js:5495)
    at amp-ads.firebaseapp.com/dist/amp.js:19105
    at <anonymous>
```

/cc @choumx @dvoytenko

**Comment by choumx** [comment]
Blocked by #7779.

**Comment by dvoytenko** [comment]
@choumx This looks like a duplicate of  #7779? If so, let's close it out?

**Comment by choumx** [comment]
This should work now. Tested on `https://localhost:8000/a4a/examples/bind/basic.amp.html`.
